python/ray/experimental/state/common.py

```diff
@@ -0,0 +1,43 @@
+import logging
+
+from dataclasses import dataclass
+
+from ray.dashboard.modules.job.common import JobInfo
+
+logger = logging.getLogger(__name__)
+
+
+@dataclass(init=True)
```

Comment:
assuming our input is strongly typed by Grpc, is these dataclasses still needed?